app/src/main/res/layout/topic_lessons_story_summary.xml

```diff
@@ -52,7 +52,7 @@
             android:id="@+id/story_progress_container"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:contentDescription="@{viewModel.storyProgressPercentageText}">
+            android:contentDescription="@{viewModel.contentDescriptionText}">
```

Comment:
Since this is being changed, can you also add a Robolectric/Espresso test to verify that it's computed correctly? We shouldn't be solely relying on e2e tests for regression testing.